### ISSUE TITLE
Request name as Tab title

### ISSUE THREAD

**Issue by lochstar** [Huachao/vscode-restclient#396]
Hi,

I sometimes have a number of request tabs open and they all have the generic title `Response(XXXms)`. Adding an option to use the `requestName` as the tab title, this would be helpful to identify which request is which.

I suggest something like the following:

### New Setting

`rest-client.requestNameAsTabTitle`: Show request name as the tab title. (Default is false)

### Example Request

```http
# @name get_todo
GET https://jsonplaceholder.typicode.com/todos/1
Content-Type: application/json
```

### Resulting Tab Title

![named response](https://i.imgur.com/Y0wT2ox.png)

This could potentionally be expanded to allow other request variables to be set as the title.

I'm happy to submit a PR to handle this feature, just thought I'd see if you were happy with this approach first.

**Comment by Huachao** [comment]
@lochstar what's the behavior when a request has no name defined in this case?

**Comment by lochstar** [comment]
Same as now, would just show `Response(XXXms)`.

**Comment by Huachao** [comment]
@lochstar OK, and if you want to contribute this project, please feel free to submit a PR, thanks.

**Comment by Huachao** [comment]
@lochstar this feature has been published in the latest version 0.22.1

**Comment by lochstar** [comment]
Awesome! Cheers.
